_pages/software-tools/zoom.md

```diff
@@ -49,43 +49,22 @@ like muting everyone (since there's always someone who leaves their mic open acc
 Unfortunately, the "shorter than 40 minutes" is a hard limitation on our Zoom accounts. Once your
 meeting reaches 40 minutes, it's over! [_RENEW! RENEW!_](http://scifi.stackexchange.com/questions/22080/what-did-the-crowd-think-it-took-to-win-renewal-in-carrousel) An easy (albeit annoying) way to get around this is simply for everyone to exit and re-enter the meeting, using the same link (the timer will restart).
 
-These meetings do not need to be scheduled in advance and you may hold as many as you like. 
+These meetings do not need to be scheduled in advance and you may hold as many as you like.
 
-### For meetings longer than 40 minutes (Pro tier)
+### For meetings longer than 40 minutes
 
-Let the Zoom Admins know as far in advance as possible in [#admins-zoom](https://gsa-tts.slack.com/messages/admins-zoom).
-Currently 18F only has two Pro licenses that allow for meetings longer than 40 minutes. What 
-this means to you is that longer meetings need to be scheduled and are generally a "first come, 
-first served" scenario. We have one large meeting extension for meetings with over 100 participants. You must have a pro account to have the large meeting extension.
-
-Before your scheduled meeting, the Zoom Admins will promote your account to Pro Status. If there are multiple 
-meetings scheduled that day, your account will be promoted as soon as the Pro account is available. The Zoom Admins will try to schedule 15 minutes before the requested time to allow you ample set up time.
-
-To schedule a meeting, please post your request in [#admins-zoom](https://gsa-tts.slack.com/messages/admins-zoom).
-The request should include the date, time, and applicable time zone. If your meeting is recurring, 
-please include the end date. 
-
-Please revalidate all recurring meeting requests annually in [#admins-zoom](https://gsa-tts.slack.com/messages/admins-zoom).
+Let the Zoom Admins know as far in advance as possible in [#admins-zoom](https://gsa-tts.slack.com/messages/admins-zoom), and they can upgrade you to be a [Licensed user](https://support.zoom.us/hc/en-us/articles/201363173-Account-Types). We have one [large meeting license](https://support.zoom.us/hc/en-us/articles/201362823-What-is-a-Large-Meeting-) for meetings with over 100 participants.
 
 ### Frequently Asked Questions
 
-Q: If I am hosting a Free meeting and it looks like it will go long, can you promote me to Pro? 
-
-A: No. Once your meeting has started, it will not convert to a Pro meeting without restarting the meeting. 
-That being said, Pro meetings can convert to Free 20 minutes before the meeting ends without affecting the meeting, if the license is needed elsewhere. 
+Q: If I am hosting a Free meeting and it looks like it will go long, can you promote me to a Licensed user?
```

Comment:
```suggestion
Q: If I am hosting a Free meeting and it looks like it will go long, can you promote me to a licensed user?
```

Reply:
The casing was intentional, since that's what they call that type of user.